libs/internal/console-driver/test-util/src/lib/spy-console/spy-console.service.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { SpyConsole } from './spy-console.service';
+
+let spy: SpyConsole;
+describe(SpyConsole.name, () => {
+  beforeEach(() => {
+    spy = new SpyConsole();
+  });
+
+  it('can be passed a lot of arguments', () => {
+    const hundredNumbers = Array(100)
+      .fill(undefined)
+      .map((_, index) => index + 1);
+
+    // eslint-disable-next-line prefer-spread
+    spy.debug.apply(spy, hundredNumbers);
```

Comment:
suggestion: Follow the lint rules (or adjust them if feeling strongly about going against their current settings)
```suggestion
    spy.debug(...hundredNumbers);
```